src/sage/graphs/graph_decompositions/tree_decomposition.pyx

```diff
@@ -499,14 +499,26 @@ def treewidth(g, k=None, kmin=None, certificate=False, algorithm=None, nice=Fals
 
     The PetersenGraph has treewidth 4::
 
-        sage: graphs.PetersenGraph().treewidth()
+        sage: petersen = graphs.PetersenGraph()
+        sage: petersen.treewidth(algorithm='sage')
+        4
+        sage: petersen.treewidth(algorithm='sage', certificate=True)
+        Tree decomposition: Graph on 6 vertices
+
+    The PetersenGraph has treewidth 4 (with `tdlib`)::
+
+        sage: petersen = graphs.PetersenGraph()
+        sage: petersen.treewidth(algorithm='tdlib')
```

Comment:
you will have to mark the test as optional in case it is tested by someone without tdlib installed.
It should be something like
```
sage: petersen.treewidth(algorithm='tdlib')  # optional tdlib
```
@mkoeppe can certainly tell us the right formulation.